src/assert.js

```diff
@@ -281,16 +277,12 @@ class Assert {
     const currentTest = (this instanceof Assert && this.test) || config.current;
 
     if (typeof block !== 'function') {
-      const message = 'The value provided to `assert.throws` in ' +
-        '"' + currentTest.testName + '" was not a function.';
-
-      currentTest.assert.pushResult({
+      return currentTest.assert.pushResult({
         result: false,
         actual: block,
-        message
+        message: 'The value provided to `assert.throws` in ' +
+          '"' + currentTest.testName + '" was not a function.'
       });
-
-      return;
```

Comment:
nit-pick: I'd prefer to keep this `return;` so to clarifu that throws is void. With the combined return, it reads to me that something is returned, or perhaps even that `throws()` and `pushResult` share a return responsibility.

Reply:
:+1:  done